pkg/cvo/upgradeable.go

```diff
@@ -92,8 +93,14 @@ type upgradeable struct {
 	Conditions []configv1.ClusterOperatorStatusCondition
 }
 
+// hasPassedDurationSinceTime checks if a certain amount of time specified by duration
+// has passed since time. Returns true if the duration has passed, returns false otherwise.
+func hasPassedDurationSinceTime(t time.Time, duration time.Duration) bool {
+	return t.Before(time.Now().Add(-duration))
```

Comment:
nit: brains are different, but mine finds it easier to read as:

```go
time.Now().After(t.Add(duration))
```

Reply:
Thanks, much better! I didn't think of that.

Reply:
Added in https://github.com/openshift/cluster-version-operator/commit/5aac7a3ef2934cbeae09a2632e22fe57eb20da91.